### PR TITLE
Few refactoring to follow pep_8 conventions, dict map in get_command in commands.py and requirements updated

### PR DESCRIPTION
**Pull request by Flintpop**
### Background

I found few things that could be rapidly refactored, like the endless elif statements in commands.py and the few blanks lacking in the project.

### Changes

Many blanks added, few types added in function prototypes to improve debugging, added a dictionary to remove all the elif in commands.py, and added few constants.
The requirements has been automatically updated by Pycharm. Some verifications may be needed.

### Test Plan

No tests.

### Change Safety

- [ ] I have added tests to cover my changes
- [x] I have considered potential risks and mitigations for my changes

There is no tests added but there should be no breaking changes.

### CODE REVIEW

**Comment by richbeales** [comment]
Probably a good PR, but mass refactorings like this are difficult to follow, and may be better being pulled apart into separate atomic PRs to address different issues.

**Review by norregaarden — APPROVED** [review]

File: requirements.txt
Comment (nponeccop) [inline requirements.txt:1]: Make the pinning down a separate PR

File: scripts/agent_manager.py [hunk 1]
```diff
@@ -3,6 +3,7 @@
 next_key = 0
 agents = {}  # key, (task, full_message_history, model)
 
+
```
Comment (nponeccop) [inline scripts/agent_manager.py:6]: An unnecessary whitespace change

File: scripts/ai_config.py [hunk 2]
```diff
@@ -1,6 +1,9 @@
+import os
```
Comment (nponeccop) [inline scripts/ai_config.py:1]: Unnecessary move of imports

File: scripts/ai_config.py [hunk 3]
```diff
@@ -37,7 +40,7 @@ def construct_full_prompt(self):
         # Construct full prompt
         full_prompt = f"You are {self.ai_name}, {self.ai_role}\n{prompt_start}\n\nGOALS:\n\n"
         for i, goal in enumerate(self.ai_goals):
-            full_prompt += f"{i+1}. {goal}\n"
+            full_prompt += f"{i + 1}. {goal}\n"
```
Comment (nponeccop) [inline scripts/ai_config.py:43]: This is good but submit whitespace changes apart.

File: scripts/config.py [hunk 4]
```diff
@@ -1,7 +1,9 @@
 import abc
 import os
+
```
Comment (nponeccop) [inline scripts/config.py:3]: Why these spaces?

File: scripts/config.py [hunk 5]
```diff
@@ -96,13 +99,13 @@ def set_smart_token_limit(self, value: int):
...
         self.openai_api_key = value
-    
+
     def set_elevenlabs_api_key(self, value: str):
         self.elevenlabs_api_key = value
-        
+
     def set_google_api_key(self, value: str):
         self.google_api_key = value
-
```
Comment (nponeccop) [inline scripts/config.py:105]: This change is actually in #590 already

**Review by nponeccop — CHANGES_REQUESTED** [review]
I suggest just to focus on moving the imports around (and isort compliance).

**Pull request closed without merging**